server/server.js

```diff
@@ -31,14 +31,9 @@ app.use((req, res, next) => {
   next();
 });
 
-const whitelist = ["http://localhost:3000"]
 const corsOptions = {
   origin: function (origin, callback) {
-    if (whitelist.indexOf(origin) !== -1) {
-      callback(null, true)
-    } else {
-      callback(new Error('Not allowed by CORS'))
-    }
+    callback(null, true)
```

Comment:
I need to look into this or m=possibly just get id of this redundant cors check entirely